ci/README.md

```diff
@@ -1,84 +1,83 @@
 
 CI will attempt to run if the following conditions are met:
 
-- The repository is in the PRX GitHub organization
+- The repository is in the PRX GitHub organization (public or private)
 - A commit is made to a master branch or a change is made to a pull request
 - The repository contains a `buildspec.yml` file
 - The `buildspec.yml` contains the string `PRX_`
 
 Beyond this, there are no technical requirements necessary to be compatible with the CI system. It will happily run any buildspec that it sees, making it possible to write an entirely custom build process for a project.
 
-In general, though, most projects will want to handle parts of the build process in similar ways. By adhering to certain conventions and including shared code libraries, much of the work needed to get an app to play nicely with the continuous deployment system will be much easier. This primarily impacts the end of the build process, where any built code or code artifacts are published somewhere that CD can deploy from.
+In general, though, most projects will want to handle some parts of the build process in similar ways. By adhering to certain conventions and including shared code libraries, much of the work needed to get an app to play nicely with the continuous deployment system will be much easier. This primarily impacts the end of the build process, where any built code or code artifacts are published somewhere that CD can deploy from.
 
-If you opt in to using the common CI support code, the project must execute the `post_build.sh` script that is maintained in the Infrastructure repo. This is the code responsible for handling the common tasks associated with prepping apps for CD. If you choose to have this script publish the code resulting from the build process (which generally will be the case), the `PRX_CI_PUBLISH` environment variable must be set to the string value `true` by the time the post_build script runs. The `post_build` script also sends a message to an SNS topic, including information about the build process and any published code. There is a callback Lambda function that also runs as part of the CI process that can take these messages, and trigger updates to the CD system.
+If you opt in to using the common CI support code, the project must execute the `post_build.sh` script that is maintained in the Infrastructure repo. This is the code responsible for handling the common tasks associated with prepping apps for CD. If you choose to have this script publish the code resulting from the build process (which generally will be the case) the `PRX_CI_PUBLISH` environment variable must be set to the string value `true` by the time the post_build script runs. The `post_build` script also sends a message to an SNS topic, including information about the build process and any published code. There is a callback Lambda function that also runs as part of the CI process that can take these messages, and trigger updates to the CD system.
```

Comment:
and `PRX_CI_PUBLISH` will have been set to `'true'` by one of the lambdas if the project meets the CI requirements defined above, right? (This reads a little bit like developer should set that as part of buildspec)